### PR TITLE
Feat/playlist

### PR DESCRIPTION
**Pull request by manish591**
### added playlist functionality

* create new playlist
* delete playlist
* add video to playlist
* delete full playlist
* delete single video from playlist

https://envpno.csb.app/explore

### CODE REVIEW

**Comment by manish591** [comment]
> _ No description provided. _

File: src/hooks/useAppServices.js [hunk 1]
```diff
@@ -155,6 +155,113 @@ const useAppServices = () => {
...
+          }
+          return item;
+        });
+        stateDispatch({
+          type: ACTION_TYPES.GET_PLAYLIST_DATA,
+          payload: { myPlaylist: arr },
+        });
+      }
+    } catch (err) {
+      console.log(err);
```
Comment (romabulani) [inline src/hooks/useAppServices.js:254]: You can use console.error here, in all catch statements

**Review by romabulani — APPROVED** [review]
- Code looks clean and modular
- Good to merge

File: src/hooks/useAppServices.js (hunk 1 above)
Comment (manish591) [reply src/hooks/useAppServices.js:254]: I forgot to do this, will add it

File: src/pages/playlist/PlaylistListing.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await axios.get(`/api/user/playlists/${playlistId}`, {
+          headers: {
+            authorization: myToken,
+          },
+        });
+        if (res.status === 200) {
```
Comment (Logan1x) [inline src/pages/playlist/PlaylistListing.jsx:25]: auth controller in some cases returning 201 status codes also. keep that in mind next time :)

**Review by Logan1x — APPROVED** [review]
Everything else looks good to me.

File: src/pages/playlist/PlaylistListing.jsx (hunk 2 above)
Comment (manish591) [reply src/pages/playlist/PlaylistListing.jsx:25]: ok

**Pull request merged**